docs/README.md

```diff
@@ -1034,6 +1035,71 @@ This message can also be appended to the default message:
           append: true
 ```
 
+### send-teams-notification
+
+Sends a teams notification with a pre-defined payload.
+
+```yml
+      uses: Alfresco/alfresco-build-tools/.github/actions/send-teams-notification@ref
+      with:
+        webhook-url: ${{ secrets.MSTEAMS_WEBHOOK }}
+```
+
+The above webhook URL is a mandatory parameter. Make sure to [Create Incoming Webhooks](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=dotnet) before using this action. Add the webhook URL as a `secret` at the repo level.
+
+Sample of a SUCCESS notification on a `push` event.
+
+![Teams Success](./images/teams-success.png)
+
+Sample of a FAILURE notification on a `push` event.
+
+![Teams Failure](./images/teams-failure.png)
+
+Below is the detailed description of the above message card.
+
+- The `summary` of the message card tells you the author, event name, and the repo where it was performed.
+- The below `change log` will fetch the commit details.
+- The `build_and_test` and `publish` are the job names added as a `need`. If you want to display needs status, you need to explicitly send the JSON object of the `needs` context as an input.
+
+  ```yml
+      uses: Alfresco/alfresco-build-tools/.github/actions/send-teams-notification@ref
+      with:
+        webhook-url: ${{ secrets.MSTEAMS_WEBHOOK }}
+        needs: ${{ toJson(needs) }}
```

Comment:
I am not getting this `needs` variable where come from, it's not a default one?

Reply:
Yes, its the [needs-context](https://docs.github.com/en/actions/learn-github-actions/contexts#needs-context) object provided by Github itself. You need to pass it from the workflow itself as its not available with in the action. Its an optional `input` that will show the status of the jobs at a glance in a workflow in the Teams message card.

Reply:
interesting, it's the first time I see it used 👍🏻